Dockerfile

```diff
@@ -35,8 +35,10 @@ COPY --from=builder /app/rhobs-synthetics-api ./
 COPY entrypoint.sh ./
 
 # Create a data directory for local storage if needed and set permissions
+# Make files executable by group and others to support OpenShift arbitrary UIDs
 RUN mkdir -p /home/rhobs/data && \
-    chown -R rhobs:rhobs /home/rhobs && \
+    chown -R rhobs:0 /home/rhobs && \
```

Comment:
Not sure I follow why we need root for the group ownership? 

Is this maybe more an issue because the `COPY` command is running under root and dropping root files on disk? 

Do we maybe just move the `USER rhobs` before the `COPY` commands?

Reply:
FYI: This is how the filesystem looks from the useradd/groupadd commands:
```
[root@7180a9268ff1 /]# groupadd -g 1001 rhobs
[root@7180a9268ff1 /]# useradd -u 1001 -g 1001 -m -d /home/rhobs -s /bin/sh rhobs
[root@7180a9268ff1 /]# ls -al /home/rhobs/
total 12
drwx------. 2 rhobs rhobs  62 Aug 20 16:48 .
drwxr-xr-x. 1 root  root   19 Aug 20 16:48 ..
-rw-r--r--. 1 rhobs rhobs  18 Feb 15  2024 .bash_logout
-rw-r--r--. 1 rhobs rhobs 141 Feb 15  2024 .bash_profile
-rw-r--r--. 1 rhobs rhobs 492 Feb 15  2024 .bashrc
[root@7180a9268ff1 /]# ls -ld /home/rhobs/
drwx------. 2 rhobs rhobs 62 Aug 20 16:48 /home/rhobs/
```